Src/Contract/AllTrustingCertificateValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.IdentityModel.Selectors;
+using System.Security.Cryptography.X509Certificates;
+
+namespace SolarWinds.InformationService.Contract2
+{
+    class AllTrustingCertificateValidator : X509CertificateValidator
```

Comment:
See **UsernameCredentials.cs** for usage of this class.  Is there any way we can perform more robust validation here, given that many/most Orion installations will be using a self-signed certificate that won't be trusted by default on a remote machine?

Reply:
I think the best we could do is an SSH-style approve-thumbprint-on-first-connect flow. But for now I am satisfied as long as the result is not worse than the current version of SwisPowerShell.